ccinput/constants.py

```diff
@@ -492,6 +493,7 @@ class CalcType(Enum):
     "dlpnoccsdt": [],
     "dlpnoqcisd": [],
     "dlpnoqcisdt": [],
+    "scf": [],
```

Comment:
Since `scf` triggers a Hartree-Fock calculation, it should be instead added as synonym of `hf`

Reply:
Done.